src/apis/club/postJoinClub.ts

```diff
@@ -0,0 +1,15 @@
+import { axiosClientWithAuth } from '../axiosClient';
+
+type JoinClubResponse = { clubId: string };
+
+const postJoinClub = async (inviteCode: string): Promise<JoinClubResponse> => {
+  const { data } = await axiosClientWithAuth.post(`clubs/invite/${inviteCode}`, {
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+  });
+
+  return data;
+};
```

Comment:
```suggestion
const postJoinClub = async (inviteCode: string) => {
  const { data } = await axiosClientWithAuth.post<JoinClubResponse>('clubs/invite/${inviteCode}`, {
    headers: {
      'Content-Type': 'application/x-www-form-urlencoded',
    },
  });

  return data;
};
```
p3;
이렇게 하면 리턴타입을 지정할 수 있어요! 함수 시그니처에 직접 타입을 선언하신 이유가 따로 있으신가요?

Reply:
엥 그러게요...?